python/packages/nisar/products/insar/GUNW_writer.py

```diff
@@ -148,6 +148,20 @@ def add_parameters_to_procinfo_group(self):
         """
         RUNWWriter.add_parameters_to_procinfo_group(self)
 
+        # Get the number of looks for both unwrapped and wrapped interferogram
+        proc_cfg = self.cfg["processing"]
+        wrap_igram_range_looks = proc_cfg["crossmul"]["range_looks"]
+        wrap_igram_azimuth_looks = proc_cfg["crossmul"]["azimuth_looks"]
+        unwrap_rg_looks = proc_cfg["phase_unwrap"]["range_looks"]
+        unwrap_az_looks = proc_cfg["phase_unwrap"]["azimuth_looks"]
+
+        unwrap_igram_range_looks = wrap_igram_range_looks
+        unwrap_igram_azimuth_looks = wrap_igram_azimuth_looks
+
+        if (unwrap_az_looks != 1) or (unwrap_rg_looks != 1):
+            unwrap_igram_range_looks = unwrap_rg_looks
+            unwrap_igram_azimuth_looks = unwrap_az_looks
```

Comment:
```suggestion
        if (unwrap_az_looks != 1) or (unwrap_rg_looks != 1):
            unwrap_igram_range_looks = unwrap_rg_looks
            unwrap_igram_azimuth_looks = unwrap_az_looks
        else:
            unwrap_igram_range_looks = wrap_igram_range_looks
            unwrap_igram_azimuth_looks = wrap_igram_azimuth_looks
```
Just a little nitpick - these variables are always going to be assigned either way, this just organizes the assignments in an if/else block for fast reading of the logic